integration_test.go

```diff
@@ -0,0 +1,373 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Remove year in copyright

Reply:
I have removed it [in this commit](https://github.com/Armanbqt/ion-go-1/pull/3/commits/3c3a578b783c1f2f1410ae0653caf7141d05ebc7#diff-0ec7e5175893797c5a6e8a7d88770a97L2)